bin/all_sky_search/pycbc_plot_kde_vals

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+import h5py, argparse
+import matplotlib.pyplot as plt
+from matplotlib.colors import LogNorm
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('--signal-file')
+parser.add_argument('--template-file', required=True)
+parser.add_argument('--param', nargs='+', required=True, help='mchirp, eta, chi_eff')
+parser.add_argument('--plot-type', choices=['kde_vs_param', 'param_vs_param'])
+parser.add_argument('--which-kde', choices=['signal_kde','template_kde', 'ratio_kde'])
+parser.add_argument('--plot-dir', required=True)
+parser.add_argument('--verbose', action='count')
+args = parser.parse_args()
+
+signal_dataset = h5py.File(args.signal_file, 'r')
```

Comment:
If the user doesn't want to plot the signal kde or ratio, it should not be necessary to read the signal file.  Also, it's a bit confusing to create a hdf file object and call it 'dataset'